src/cli/index.ts

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env node
+import React from "react";
+import { AppProps, render } from "ink";
+import { Command } from "commander";
+
+import App from "./ui/ui";
+const program = new Command();
+
+program
+  .name("jslings")
+  .description("jslings can help YOU master Javascript on the terminal")
+  .on("--help", () => {
+    console.log("");
+    console.log("Examples:");
+    console.log("");
+    console.log("  $ jslings watch");
+    console.log("  $ jslings hint");
+  });
+
+program
+  .command("watch")
+  .alias("w")
+  .description("jslings interactive code testing UI")
+  .option("-t, --test <testName>", "path to the test")
+  .action(async ({ test }: Command) => {
```

Comment:
We should check if they passed in `test`, if not, find out which one they are on.

Reply:
Do you mean that it not provided the -t flag, just watch the current test to complete? 

I'm thinking we can have a `--all` flag to see the tests available, which goes in `utils`.

Reply:
Yeah. For example `rustlings` doesn't require you to pass the test. If you `rustlings watch`, you just pick up where you left off. I think it'd be great if we mimic that functionality! Makes it easier for the user to keep going. 🏃

Reply:
I see, yeah I think this will go in utils tbh, will try this out

Reply:
@saphal1998 we can just create an issue for this and do it in a different PR!

Reply:
Yeah, I was just brainstorming this, and it seems like we would need to add a bunch of stuff to do this. Will make a seperate PR for this